storyblok/storyblok-design-system
# feat(icon): introduce webhook internal icon

Introduces the `webhook` icon

## Pull request type

Jira Link: [WORK-168](https://storyblok.atlassian.net/browse/WORK-168)




## How to test this PR



Run storybook and make sure a nem icon named `webhook` is being displayed properly in the Components > SbIcon > All Icons section

## What is the new behavior?



- A new icon named `webhook` must be available for use

## Other information
